packages/metrics/src/features/registration/handler.ts

```diff
@@ -99,6 +99,8 @@ export async function registrarRegistrationWaitingExternalValidationHandler(
 ) {
   const points = []
 
+  await createUserAuditPointFromFHIR('SENT_FOR_APPROVAL', request)
+
```

Comment:
`registrarRegistrationWaitingExternalValidationHandler` is not a 'SENT_FOR_APPROVAL' event so we need to remove this